src/Illuminate/Foundation/Console/BugCommand.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Illuminate\Foundation\Console;
+
+use Closure;
+use Illuminate\Console\Command;
+use Illuminate\Support\Composer;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Str;
+use Symfony\Component\Console\Attribute\AsCommand;
+
+#[AsCommand(name: 'bug')]
+class BugCommand extends Command
+{
+    /**
+     * The console command signature.
+     *
+     * @var string
+     */
+    protected $signature = 'bug';
```

Comment:
Why do we need it if we have class Attribute with command name already?

Reply:
What's your suggestion?

Reply:
Probably the idea is to remove the attribute and keep the signature or vice versa.

Reply:
We want both: https://github.com/laravel/framework/pull/50617